pusher-api/user-guides/create-subscriptions.md

```diff
@@ -172,6 +173,16 @@ To verify that events come from Zettle, calculate a signature and compare it wit
 
 2. Compare the newly calculated signature with the value in the HTTP header `X-iZettle-Signature` and take actions accordingly.
 
+## Step 5: Avoid webhook requests timeout
```

Comment:
I suggest changing the step title to "Configure request timeout", since this is what you actually do in this step.

Reply:
I changed it to "Prevent xxx", since we can't change the timeout itself.